skcosmo/decomposition/kpcovr.py

```diff
@@ -453,9 +453,9 @@ def score(self, X, Y):
         K_VV = self._get_kernel(X)
 
         if self.center:
-            K_NN = self.centerer_.transform(K_NN)
-            K_VN = self.centerer_.transform(K_VN)
-            K_VV = self.centerer_.transform(K_VV)
+            K_NN = self._centerer.transform(K_NN)
+            K_VN = self._centerer.transform(K_VN)
+            K_VV = self._centerer.transform(K_VV)
```

Comment:
```suggestion
            K_NN = self.centerer_.transform(K_NN)
            K_VN = self.centerer_.transform(K_VN)
            K_VV = self.centerer_.transform(K_VV)
```
Since the centerer is fit during the `fit` call, it should have a trailing underscore by sklearn naming guidelines.
Lines 314--315 should also have trailing underscores.

Reply:
So this was a bug I saw before -- the `centerer`'s in KPCovR were named differently (`centerer_` in some places, `_centerer` in others. No qualms with changing which it is, just making sure it's consistent.